### PR TITLE
SPA Navigation

### PR DESCRIPTION
**Pull request by sja-thedude**
- Added Books list.
- Added book form.
- Added Contact info.
- the current date is added below the navigation bar. 
- Javascript file has been updated
- CSS File has been Update
- Html File has been updated

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```
Comment (mikenath223) [inline script.js:1]: - [ ] Kindly remove this eslint disable rule it's best to fix the issue in the code.

File: index.html [hunk 2]
```diff
@@ -32,6 +50,22 @@ <h1 class="heading">Add a new book</h1>
...
 
+    <div class="contactsection">
+        <h1 class="heading">Contact Information</h1>
+        <p>Do you have any questions or you just want to say "Hello"?<br> You can reach out to us</p>
+        <ul class="contactul">
+          <li>Email: sja.affu765@gmail.com</li>
+          <li>Phone Number: 00966572705741</li>
+          <li>Address: Jeddah ,KSA</li>
+        </ul>
+    </div>
```
Comment (mikenath223) [inline index.html:61]: - [ ] OPTIONAL - Consider adding margin and padding to elements, it's important elements have spacing between them. Most contents on the page are pretty crowded together.

**Review by Lameck1 — APPROVED** [review]
Hi, @sja-thedude & @1GORDON,

Well done on working on the previously requested changes 👏🏼.

Please allow to list these as general suggestions to your project:
Looking at the anomalies displayed in the images below, the sense of building a Single Page App is no longer realizable.

| **_The Add New Book page_** | **_The Contact Page_** |
| --- | --- |
| Add_New_Page | contact_page |

The book list is not supposed to be visible on the above-highlighted pages. I am sure this is something small you can figure out and work on.

Your project is complete!  It's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**